### PR TITLE
Add opt in fetch

### PR DESCRIPTION
**Pull request by comfroels**

### CODE REVIEW

File: generators/app/templates/webpack.config.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ if (process.env.INCLUDE_WEBPACK_HTML) {
 
 module.exports = {
   entry: [
+    <%- withFetch ? "'whatwg-fetch'," : '' %>
```
Comment (kendaleiv) [inline generators/app/templates/webpack.config.js]: Will this create an empty string item when the user doesn't wish to include `whatwg-fetch`? Although, looking at it closer, it looks like that might not be the case.

Will it create a blank line though? :smile:

Comment (comfroels) [reply generators/app/templates/webpack.config.js]: It does. Right now it leaves a blank whitespace, which doesn't interfere with anything running. However I couldn't figure out with ejs how to get rid of the whitespace if we don't have the variable. There is a closing tag `-%>` that trims the next newline I think. But I can't really do that conditionally?

Comment (comfroels) [reply generators/app/templates/webpack.config.js]: By blank whitespace I mean a blank line

Comment (comfroels) [reply generators/app/templates/webpack.config.js]: I might have a solution... hold on

Comment (comfroels) [reply generators/app/templates/webpack.config.js]: Boom, got it. Looks a little elementary but works and doesn't leave that blank line.  💣

**Pull request merged**